dv/uvm/icache/data/common_sim_cfg.hjson

```diff
@@ -0,0 +1,128 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  project:          opentitan
+  doc_server:       docs.opentitan.org
+  results_server:   reports.opentitan.org
+
+  flow:             sim
+  flow_makefile:    "{proj_root}/dv/uvm/icache/data/sim.mk"
+
+  import_cfgs:      ["{proj_root}/dv/uvm/icache/data/common_project_cfg.hjson",
+                     "{proj_root}/dv/uvm/icache/data/common_modes.hjson",
+                     "{proj_root}/dv/uvm/icache/data/fusesoc.hjson",
+                     "{proj_root}/dv/uvm/icache/data/{tool}/{tool}.hjson"]
+
+  // Default directory structure for the output
+  build_dir:          "{scratch_path}/{build_mode}"
+  run_dir_name:       "{index}.{test}"
+  run_dir:            "{scratch_path}/{run_dir_name}/out"
+  sw_build_dir:       ""
+  sw_root_dir:        ""
+
+  // pass and fail patterns
+  pass_patterns:    ["^TEST PASSED (UVM_)?CHECKS$"]
+  fail_patterns:    ["^UVM_ERROR\\s[^:].*$",
+                     "^Assert failed: ",
+                     "^\\s*Offending '.*'",
+                     "^TEST FAILED (UVM_)?CHECKS$"]
+
+  // Default TileLink widths
+  tl_aw: 32
+  tl_dw: 32
+  tl_dbw: 4
+
+  // Default UVM verbosity settings
+  n: UVM_NONE
+  l: UVM_LOW
+  m: UVM_MEDIUM
+  h: UVM_HIGH
+  d: UVM_DEBUG
+
+  // Default waves dump settings
+  dump_file:  waves.{dump}
+
+  // Top level simulation entities.
+  sim_tops: ["-top {tb}"]
+
+  // Default build and run opts
+  build_opts: [// List multiple tops for the simulation
+               "{sim_tops}",
+               // Standard UVM defines
+               "+define+UVM",
+               "+define+UVM_NO_DEPRECATED",
+               "+define+UVM_REGEX_NO_DPI",
+               "+define+UVM_REG_ADDR_WIDTH={tl_aw}",
+               "+define+UVM_REG_DATA_WIDTH={tl_dw}",
+               "+define+UVM_REG_BYTENABLE_WIDTH={tl_dbw}"]
+
+  run_opts: ["+UVM_NO_RELNOTES",
+             "+UVM_VERBOSITY={verbosity}"]
+
+  // Default list of things to export to shell
+  exports: [
+    DUMP_FILE: {dump_file}
+    WAVES: {waves}
+    DUT_TOP: {dut}
+    TB_TOP: {tb}
+  ]
+
+  // Build modes are collection of build_opts and run_opts
+  // A test can enable a specific build mode by setting 'use_build_mode' key
+  build_modes: [
+    {
+      name: foo
+      build_opts: ["+define+bx",
+                   "+define+by",
+                   "+define+bz"]
+      run_opts: ["+rx=1",
+                 "+ry=2",
+                 "+rz=3"]
+    }
+    {
+      name: bar
+      build_opts: ["+define+bbaru1",
+                   "+define+bbaru2",
+                   "+define+bbaru3"]
+      run_opts: ["+rbar1u=1",
+                 "+rbar2u=2",
+                 "+rbar3u=3"]
+    }
+  ]
```

Comment:
You can get rid of this - this is just an example of how you can set 'build_modes'.